libs/hacl/CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ endif()
 # ---- Library Sources ----
 include(${PROJECT_SOURCE_DIR}/libs/hacl/sources.cmake)
 
-if(ELECTIONGUARD_BUILD_SHARED)
+if(BUILD_SHARED_LIBS)
```

Comment:
when building the downstream dependencies `ELECTIONGUARD_BUILD_SHARED` is not set since it is set as part of the electionguard target. this instead uses the standard flag